ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -369,18 +369,29 @@ Resources:
             'Fn::Sub': '${AWS::Region}-sc-enduser-iam-ServiceCatalogEndusers-RoleId'
 Outputs:
   LinuxInstanceId:
+    Description: 'The ID of the EC2 instance'
     Value: !Ref 'LinuxInstance'
   LinuxInstancePrivateIpAddress:
+    Description: 'The IP Address of the EC2 instance'
     Value: !GetAtt 'LinuxInstance.PrivateIp'
   EC2InstanceType:
+    Description: 'The EC2 instance type'
     Value: !Ref 'EC2InstanceType'
   IAMInstanceRole:
+    Description: 'The IAM Role associated with the instance'
```

Comment:
do we even need to output IAMInstanceRole & IAMInstanceProfile?  i don't think it really has any meaning to a SC user?

Reply:
Totally agree. I will remove those outputs.